src/app/extensions/quoting/shared/quoting-basket-line-items/quoting-basket-line-items.component.ts

```diff
@@ -0,0 +1,36 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { groupBy } from 'lodash-es';
+import { Observable, map } from 'rxjs';
+
+import { CheckoutFacade } from 'ish-core/facades/checkout.facade';
+import { LineItem } from 'ish-core/models/line-item/line-item.model';
+import { GenerateLazyComponent } from 'ish-core/utils/module-loader/generate-lazy-component.decorator';
+
+import { QuotingFacade } from '../../facades/quoting.facade';
+
+@Component({
+  selector: 'ish-quoting-basket-line-items',
+  templateUrl: './quoting-basket-line-items.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+@GenerateLazyComponent()
+export class QuotingBasketLineItemsComponent implements OnInit {
+  lineItems$: Observable<[string, LineItem[]][]>;
+
+  constructor(private checkoutFacade: CheckoutFacade, private quotingFacade: QuotingFacade) {}
+
+  ngOnInit() {
+    this.quotingFacade.quotingEntities$();
+    this.lineItems$ = this.checkoutFacade.basket$.pipe(
+      map(basket => Object.entries(groupBy(basket.lineItems, 'quote')))
+    );
+  }
+
+  getName(quoteId: string) {
+    return this.quotingFacade.name$(quoteId);
+  }
+
+  onDeleteQuote(lineItems: LineItem[]) {
+    lineItems.forEach(item => this.checkoutFacade.deleteBasketItem(item.id));
```

Comment:
It looks like we shouldn't use multiple deletes here, as the first delete deletes all quote items from the basket already, and the subsequent deletes fail.
@SGrueber Is there a better way to delete quotes from the basket that via items? I did not really find one in the API doc.

Reply:
Maybe for now it is best to just delete the first line item.

Reply:
This is the way to go. If a basket item is deleted that is based on a quote all other basket items of this quote will also be deleted by this REST call.